src/ert/_c_wrappers/enkf/enkf_main.py

```diff
@@ -144,6 +179,21 @@ def _generate_parameter_files(
 
         node = ens_config[key]
         type_ = node.getImplementationType()
+
+        if type_ == ErtImplType.FIELD:
```

Comment:
Could move this down to live 167 to have a smaller diff

Reply:
I think this might have been improved in the rebase